os/binfmt/binfmt_execsymtab.c

```diff
@@ -130,7 +130,7 @@ void exec_getsymtab(FAR const struct symtab_s **symtab, FAR int *nsymbols)
 	 * size are returned as a single atomic operation.
 	 */
 
-	flags = irqsave();
+	flags = enter_critical_section();
```

Comment:
Do you change all of irqsave to enter_..._section in this PR? Should we use it from now?

Reply:
Actually, it is not totally clear to me either. 
When we check nuttx code, both irqsave and enter_critical_section are being used. 
So, where ever nuttx is using enter_critical_section, we have done the same in tizenrt. In all other places, we continue to use irqsave(). 

However, I think we need to investigate more on this. But, it will not impact the current PR. Since, the basic SMP operation is working well, I think we can go ahead with this PR. If changes are required for irqsave, we will raise it in a new PR.

Reply:
Hi @kishore-sn ,

May I know which is the suitable scenario to apply enter_critical_section, and which is the case of irqsave(), with SMP config enabled?